packages/opencrvs/test/testData.js

```diff
@@ -0,0 +1,551 @@
+export const birthRecordData = [
```

Comment:
There is a LOT of data in this file and it's hardly used by any of the tests. Do we really need to save so much to the repo? If we do, can we make sure that tests are actually using it?